Private/Send-Notification.ps1

```diff
@@ -61,15 +58,13 @@ function Send-Notificaton {
                     -MessageTitle $MessageTitle `
                     -Color $Color `
                     -Sections $Section1 `
-                    -Supress $false #`
-                # -Verbose
-                Write-Color @script:WriteParameters -Text "[i] Teams output: ", $Data -Color White, Yellow
+                    -Supress $false
+                $Logger.AddRecord("Teams output: $Data")
```

Comment:
Modify example please to add logger. You only added it for main example. The one using teams, slack is using a different one.